get/releases.go

```diff
@@ -0,0 +1,94 @@
+package get
+
+import (
+	"context"
+	"os"
+	"sort"
+	"strconv"
+	"text/tabwriter"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+)
+
+type releasesCmd struct {
+	Name string `arg:"" help:"Name of the Application for which to get the Releases. If omitted all Applications will be listed." default:""`
+}
+
+func (cmd *releasesCmd) Run(ctx context.Context, client *api.Client, get *Cmd) error {
+	releaseList := &apps.ReleaseList{}
+
+	if err := get.list(ctx, client, releaseList, matchApplicationName(cmd.Name)); err != nil {
+		return err
+	}
+
+	if len(releaseList.Items) == 0 {
+		printEmptyMessage(apps.ReleaseKind, client.Namespace)
+		return nil
+	}
+
+	switch get.Output {
+	case full:
+		return printReleases(releaseList.Items, get, true)
+	case noHeader:
+		return printReleases(releaseList.Items, get, false)
+	}
+
+	return nil
+}
+
+// prepareReleasesOutput prepares Release collection for printing
+func prepareReleasesOutput(releases []apps.Release) map[string][]apps.Release {
+	orderedReleases := make(map[string][]apps.Release)
+
+	for _, release := range releases {
+		orderedReleases[release.ObjectMeta.Labels[util.ApplicationNameLabel]] =
+			append(orderedReleases[release.ObjectMeta.Labels[util.ApplicationNameLabel]], release)
```

Comment:
Nit: assignment over two lines looks a bit strange, maybe put `release.ObjectMeta.Labels[util.ApplicationNameLabel]` in a variable so this line can be shorter?